packages/frontend/pages/lobby.tsx

```diff
@@ -34,8 +46,15 @@ const Lobby = (props: IProps): ReactElement => {
           <Grid.Row className="taako_wrapper">
             <Taako />
           </Grid.Row>
-          <Grid.Row>Announcements</Grid.Row>
-          <Grid.Row>Latest avatars</Grid.Row>
+          <Grid.Row className="taako_wrapper">
```

Comment:
Please call this announcements_wrapper or something, and then assign it to the same css rules as taako_wrapper.

Reply:
I don't need the taako_wrapper classname here actually. Should I remove it?

Reply:
Ah yeap, I forgot to remove it when I moved it into its own function. Please remove it :)